### DISCUSSION TITLE
Use of an option I don't understand which function it has…

### DISCUSSION THREAD

**Discussion by bledu** [mrkai77/Loop#236] [Q&A] [answered]
Hello,  I'm manager of a French weekly digital Mac magazine (VVMac hebdo) and I'm writing an article about Loop in our next issue. I have just one small problem : I don't understand what  does the option Settings > General > Animate windows being resized. Activated or not, I can't see any visual thing happening differently. There is nothing on the GitHub page to explain this option. Could you tell me what it is supposed to do, how to use it (it is used with the radial menu or not ? with the Windows Snapping ?)
Thank you. you can answer to me on my email address : bledu@vvmac.com (as I don't go regularly on GitHub).

**Comment by mrkai77** [comment, marked as answer]
It animates windows as they're being resized!
Example:
https://github.com/MrKai77/Loop/assets/68963405/752574dd-9797-40ff-ac52-eeb380ca866a
As opposed to:
https://github.com/MrKai77/Loop/assets/68963405/bbe724f2-7335-415c-8f24-a2f771e0def6

However, this is a beta feature so it will not work correctly with all apps, and may be removed in any future release if it stops working :)

**Reply by bledu** [reply]
OK Thank you. 
The article will be published Saturday the 16th of march. If you would like to get it, I could sent you the PDF. 
Just give me an address to send to. Have a nice weekend.
